src/Microsoft.Data.Analysis/DataFrame.IO.cs

```diff
@@ -23,6 +23,12 @@ private static Type GuessKind(int col, List<string[]> read)
                     throw new FormatException(string.Format(Strings.LessColumnsThatExpected, nbline + 1));
 
                 string val = line[col];
+
+                if (val == "null" || val == "Null")
```

Comment:
```suggestion
                if (val.Equals("null", StringComparison.OrdinalIgnoreCase))
```
Though I don't have a strong opinion on this...
It would also handle `NULL`, so maybe it's a bit safer.

Reply:
Yup. Good idea.